package-overrides/npm/angular@1.5.0-rc.2.json

```diff
@@ -0,0 +1,14 @@
+{
+  "format": "global",
+  "jspmNodeConversion": false,
+  "main": "angular",
+  "files": [
+    "angular.js",
+    "package.json"
+  ],
```

Comment:
It's best not to include this, as otherwise it won't be obvious why the other files are missing.